src/TaCo/connection-dialog.tcd

```diff
@@ -0,0 +1,14 @@
+<connection-dialog class='odfe_sql_odbc'>
+  <connection-config>  
+    <authentication-mode value='ComboBoxIntegrated' />
+        <authentication-options>
+            <option name="None" value="auth-none" default="true" />
```

Comment:
Is it possible to set this value to `NONE` so we can directly parse the value as our connection string `Auth` parameter?
```suggestion
            <option name="None" value="NONE" default="true" />
```

Reply:
done